lib/generators/arclight/install_generator.rb

```diff
@@ -55,5 +55,9 @@ def add_repository_config
     def add_download_config
       copy_file 'config/downloads.yml' unless File.exist?('config/downloads.yml')
     end
+
+    def modify_blacklight_yml
+      gsub_file 'config/locales/blacklight.yml', "(^.*)\n(.*)\n(.*)", "\1\n\2\n    application_name: 'Arclight'"
```

Comment:
Can we just replace `application_name: 'Blacklight'` with `application_name: 'ArcLight'` instead of using the regular expression?